### PR TITLE
Seeding DB and API calls

### PR DESCRIPTION
**Pull request by djian001**
npm install & start react-dev, server

### CODE REVIEW

**Comment by kimberlyjk** [comment]
in order to seed DB need to run `node database/seedScript.js`

File: database/index.js [hunk 1]
```diff
@@ -12,9 +12,33 @@ db.once('open', () => {
 })
 
 const reviewSchema = new mongoose.Schema({
-
+  versionKey: false,
+  campId: {
+    unique: true,
+    type: Number,
+    sparse: true
```
Comment (kimberlyjk) [inline database/index.js:20]: this is interesting. I don't really know what it does. lol I'll look into it

File: database/seedScript.js [hunk 2]
```diff
@@ -1,25 +1,79 @@
...
     return Math.floor(Math.random() * max);
   }
 
   let randReviews = Math.floor(Math.random() * 10);
 
-  let ownerReponse = [null, faker.lorem.sentence()]
-  let randResponse = ownerReponse[Math.floor(ownerReponse.length * Math.random())];
-
-  for (var id = 0; id <= 10; id++) {
+  for (var id = 1; id <= 10; id++) {
```
Comment (kimberlyjk) [inline database/seedScript.js:74]: should be up to 100?

File: client/src/app.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      reviews: []
+    }
+  }
+
+  componentDidMount() {
+    $.ajax({
+      url: '/reviews',
```
Comment (kimberlyjk) [inline client/src/app.jsx:15]: This call works, but doesn't match what is shown in the App and Service plan. I would work on getting those calls to match.. but for now it does call you server and server serves up a response!

**Review by kimberlyjk — APPROVED** [review]
Minor changes but its mostly due to the nature of getting this project moving along. 

Great work Denise!! you deserve a dan-tat.

File: database/seedScript.js (hunk 2 above)
Comment (djian001) [reply database/seedScript.js:74]: Yes, was testing with 10!

File: client/src/app.jsx (hunk 3 above)
Comment (djian001) [reply client/src/app.jsx:15]: will be updating soon.

File: database/index.js (hunk 1 above)
Comment (djian001) [reply database/index.js:20]: I'm not 100% sure either.

**Pull request merged**